### ISSUE TITLE
The meaning of the position map ?

### ISSUE THREAD

**Issue by rakacendekia** [YadiraF/PRNet#198]
I would like to ask, what do the colors on the position map mean and how do they affect the face reconstruction process? Does it mean the same as in PNCC?
![IBUG_image_008_1_0_posmap](https://user-images.githubusercontent.com/72121999/96213234-b0303500-0fa2-11eb-861c-7bdc29eeaf6e.jpg)

**Comment by Suvi-dha** [comment]
Yes, its same as pncc that is converting normalized x,y,z values to r,g,b values. The difference is that here x, y, z coordinates in 3d space is mapped to texture coordinates (vt_x, vt_y) associated with 3d model and then this map is plotted where x, y, z values are mapped to r,g,b values at locations vt_x, vt_y. Hope this made sense.
